### DISCUSSION TITLE
json validation  String is not a URI: URI with a scheme is expected.

### DISCUSSION THREAD

**Discussion by manufv** [OAI/OpenAPI-Specification#2584] [Q&A] [answered]
hello,

I try to validate my oas 3.1.0 and I get a error on internal $ref values **String is not a URI: URI with a scheme is expected** 

I define like that 
```json
"400":	{
   "$ref":	"#/components/responses/rsp400"
}
```

/components/responses/rsp400 is correctly defined (jump to with editor works good)

To validate the json semantic, I use the "$schema": "https://spec.openapis.org/oas/3.1/schema/2021-03-02".

where $ref is defined as a #/$def/uri and uri is format 'uri' which request a uri **scheme** 
```json
    "uri": {
      "type": "string",
      "format": "uri"
    }
```

what to do to solve the error ?
thanks

**Comment by MikeRalphson** [comment]
Ping @jdesrosiers @Relequestual

**Comment by jdesrosiers** [comment, marked as answer]
The format should be `uri-reference`, not `uri`. I'll make a PR to fix it.

However, `format` should be annotation-only and not cause validation failures.

**Reply by jdesrosiers** [reply]
PR: #2586

**Comment by handrews** [comment]
Fixed years ago, closing!
